scm/pipeline.go

```diff
@@ -0,0 +1,14 @@
+package scm
+
+import "time"
+
+type (
+	Pipeline struct {
```

Comment:
remove the `json: ...` annotations from this struct for consistency with other structs

Reply:
Done